app/controllers/application_controller.rb

```diff
@@ -20,6 +20,25 @@ class ApplicationController < ActionController::Base
 
   layout 'application'
 
+  # A common method for controllers to find a Dor object, whether the
+  # input is a DRUID, a PID or a Dor object.
+  # @param druid [String|ActiveFedora::Base]
+  #              DruidTools::Druid is used to parse String, which is
+  #              then input for Dor.find to obtain a Dor object
+  #              Dor objects inherit from ActiveFedora::Base
+  # @raises ArgumentError
+  def find_druid(druid)
```

Comment:
Why is this an action in the controller? If we need this I would advocate for adding a class that contains this logic.